apollo-backend/src/test/java/io/logz/apollo/helpers/ModelsGenerator.java

```diff
@@ -53,6 +58,72 @@ public static Environment createAndSubmitEnvironment(ApolloTestClient apolloTest
         return testEnvironment;
     }
 
+    public static EnvironmentsStack createEnvironmentsStack() {
+        EnvironmentsStack testEnvironmentsStack = new EnvironmentsStack();
+        testEnvironmentsStack.setName("environments-stack-" + Common.randomStr(5));
+        testEnvironmentsStack.setEnabled(true);
+        testEnvironmentsStack.setStackType(StackType.ENVIRONMENTS);
+        testEnvironmentsStack.setEnvironments(new ArrayList<>());
+
+        return testEnvironmentsStack;
+    }
+
+    public static EnvironmentsStack createEnvironmentsStack(List<Integer> environments) {
+        EnvironmentsStack testEnvironmentsStack = createEnvironmentsStack();
+        testEnvironmentsStack.setEnvironments(environments);
+
+        return testEnvironmentsStack;
+    }
+
+    public static EnvironmentsStack createAndSubmitEnvironmentsStack(ApolloTestClient apolloTestClient) throws Exception {
+        EnvironmentsStack testEnvironmentsStack = createEnvironmentsStack();
+        int id = apolloTestClient.addEnvironmentsStack(testEnvironmentsStack).getId();
+        testEnvironmentsStack.setId(id);
+
+        return testEnvironmentsStack;
+    }
+
+    public static EnvironmentsStack createAndSubmitEnvironmentsStack(ApolloTestClient apolloTestClient, List<Integer> environments) throws Exception {
```

Comment:
`createAndSubmitEnvironmentsStackWithEnvironments`

Reply:
Having an overload that have completely different functionality based on the parameters in not a good practice